app/gui/src/main/scala/org/bitcoins/gui/dlc/DLCPaneModel.scala

```diff
@@ -156,7 +156,10 @@ class DLCPaneModel(resultArea: TextArea, oracleInfoArea: TextArea) {
                                              "dummy oracle")
 
           val announcementSig =
-            privKey.schnorrSign(CryptoUtil.sha256(oracleEvent.bytes).bytes)
+            privKey.schnorrSign(
+              CryptoUtil
+                .taggedSha256(oracleEvent.bytes, "DLC/oracle/announcement/v0")
+                .bytes)
```

Comment:
all invocations of `taggedSha256` should be replaced with a new function in `CryptoUtil` akin to 
https://github.com/bitcoin-s/bitcoin-s/blob/master/crypto/src/main/scala/org/bitcoins/crypto/CryptoUtil.scala#L71
including the per-compiled val for the tag's hash.